apis/Google.Cloud.Vision.V1/Google.Cloud.Vision.V1/ImagePartial.cs

```diff
@@ -30,6 +30,10 @@ public partial class Image
         /// which may either be a Google Cloud Storage URI or a publicly accessible HTTP or HTTPS URI. The
         /// image is fetched from the URI by the Google Cloud Vision server.
         /// </summary>
+        /// <remarks>
```

Comment:
Might be worth putting at least a hint of this in the summary, as most users probably won't see these remarks.

Reply:
Good plan. Will move it in the morning, then merge.